flink-connectors/flink-connector-elasticsearch5/src/main/resources/META-INF/NOTICE

```diff
@@ -7,10 +7,12 @@ The Apache Software Foundation (http://www.apache.org/).
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
 - com.carrotsearch:hppc:0.7.1
-- com.fasterxml.jackson.core:jackson-core:2.10.1
-- com.fasterxml.jackson.dataformat:jackson-dataformat-cbor:2.10.1
-- com.fasterxml.jackson.dataformat:jackson-dataformat-smile:2.10.1
-- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.1
+- com.fasterxml.jackson.core:jackson-core:2.10.5
+- com.fasterxml.jackson.core:jackson-databind:2.10.5.1
+- com.fasterxml.jackson.core:jackson-annotations:2.10.5
```

Comment:
these 2 are incorrect; this module (and likely others with a similar problem) is only binding jackson-core.

Reply:
huh...at least in 1.12 this actually seems to be correct...I'm wondering why that is the case...

Reply:
well this is interesting; in 2.10.1 the dataformats had a `provided` dependency on databind/annotations, but since 2.10.4 they are compile dependencies.

Reply:
Yes, I was surprised at the beginning. Later I found that starting from `2.10.4`, the scope of `jackson-dataformat-yaml`'s dependence on `jackson-databind` has changed from `provided` to `compile`.
In fact, from `2.11.0`, the scope of `jackson-dataformat-cbor` and `jackson-dataformat-smile`s dependence on `jackson-databind` has also changed from `provided` to `compile`.